checkov/github_actions/utils.py

```diff
@@ -69,3 +72,54 @@ def is_schema_valid(config: dict[str, Any] | list[dict[str, Any]]) -> bool:
             )
 
     return False
+
+
+def get_gha_files_definitions(root_folder: str | Path,
+                              files: "list[Path] | None" = None,
+                              runner_filter: RunnerFilter | None = None,) -> tuple[dict[str, Any], dict[str, Any]]:
+    definitions = {}
+    definitions_raw = {}
+    file_paths = get_scannable_file_paths(root_folder=root_folder)
+
+    for file_path in file_paths:
+        result = parse_file(f=file_path)
+        if result is not None:
+            definitions[str(file_path)] = result[0]
```

Comment:
can add examples of the 'result' to better understand what is expected to be in result[0], and result[1]

Reply:
Sure, added a clarification comment :)